test/specs/modules/Dropdown/Dropdown.spec.js

```diff
@@ -33,6 +37,61 @@ describe('Dropdown', () => {
     expect(openSpy).to.have.been.calledWith(false);
   });
 
+  it('should close the menu when clicking on option', () => {
+    const wrapper = shallow(DropdownWithRequired, {
+      propsData: {
+        options: [{ text: 'foo', value: 1 }],
+      },
+    });
+    wrapper.vm.setOpen(true);
```

Comment:
I wouldn't use a private method to test this. What about adding a function `openDropdown = (wrapper) => wrapper.trigger('click')`?